sample-app/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="icon" href="/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Process Optimizer Frontend</title>
+    <title>Brownie Bee</title>
```

Comment:
Maybe stick to Process Optimizer Frontend

Reply:
Changed